diff --git a/plugin/build/ios/appDelegate.js b/plugin/build/ios/appDelegate.js
index 33adf82..b6a30d8 100644
--- a/plugin/build/ios/appDelegate.js
+++ b/plugin/build/ios/appDelegate.js
@@ -35,8 +35,9 @@ function getAppDelegateClassClosingLine(src) {
     return classRange.endLine;
 }
 function applyIosAppDelegate(contents) {
-    let nextContents = (0, generateCode_1.replaceGeneratedContentsAtLine)({
-        src: contents,
+    let nextContents = contents.replace(/^import Expo$/m, 'public import Expo');
+    nextContents = (0, generateCode_1.replaceGeneratedContentsAtLine)({
+        src: nextContents,
         newSrc: 'import UserNotifications',
         tag: 'jpush-swift-import-usernotifications',
         getLineIndex: getLastImportLine,
@@ -218,7 +219,7 @@ const getJPushDelegateExtension = () => {
 
   // 通知设置
   @objc public func jpushNotificationCenter(_ center: UNUserNotificationCenter,
-                                           openSettingsFor notification: UNNotification?) {
+                                           openSettingsFor notification: UNNotification) {
     #if DEBUG
     print("打开通知设置")
     #endif