diff --git a/plugin/build/ios/appDelegate.js b/plugin/build/ios/appDelegate.js
index 33f44b8a4f0f4a5197c447d3cfd5b8f18fe92726..4cc2f5e1a0a4836c204cd783e8a0f329f8fbc19b 100644
--- a/plugin/build/ios/appDelegate.js
+++ b/plugin/build/ios/appDelegate.js
@@ -36,8 +36,9 @@ function getAppDelegateClassClosingLine(src) {
     return classRange.endLine;
 }
 function applyIosAppDelegate(contents) {
-    let nextContents = (0, generateCode_1.replaceGeneratedContentsAtLine)({
-        src: contents,
+    let nextContents = contents.replace(/^import Expo$/m, 'public import Expo');
+    nextContents = (0, generateCode_1.replaceGeneratedContentsAtLine)({
+        src: nextContents,
         newSrc: 'import UserNotifications',
         tag: 'jpush-swift-import-usernotifications',
         getLineIndex: getLastImportLine,
